evm/db/backends/level.py

```diff
@@ -0,0 +1,64 @@
+import shutil
+from .base import (
+    BaseDB,
+)
+
+
+class LevelDB(BaseDB):
+    db = None
+    db_path = None
```

Comment:
Is the `db_path` not derivable from the `self.db` property?  If so, I think this should be removed in favor of just pulling this value from the leveldb instance itself.